README.md

```diff
@@ -1 +1,2 @@
 # err-lastseen
+A plugin for Errbot which tells you when user was last seen in irc chat.
```

Comment:
@JakubNvk This is my bad for not being explicit enough about this, but could we remove the IRC chat from the README? If possible, I would like this to be as general as possible (and thus usable for protocols other than IRC).